src/sql/expertise_scales.sql

```diff
@@ -1,4 +1,11 @@
-SELECT 'interest' AS type, description, ranking FROM interest_scale
-UNION ALL
-SELECT 'experience' AS type, description, ranking FROM experience_scale
-ORDER BY type, ranking
+WITH interest_agg AS (
+  SELECT
+    ARRAY_AGG(ARRAY[ranking::text, description] ORDER BY ranking) AS interest
+  FROM interest_scale
+), experience_agg AS (
+  SELECT
+    ARRAY_AGG(ARRAY[ranking::text, description] ORDER BY ranking) AS experience
+  FROM experience_scale
+)
+SELECT *
+FROM interest_agg, experience_agg
```

Comment:
Since you're not doing any additional processing on the CTE queries, you can just do subselects--it'll be faster (trivially so):

Here is what the two different plans looks like:
```
expertisebot-ben=> EXPLAIN ANALYZE
expertisebot-ben-> SELECT
expertisebot-ben->   (SELECT ARRAY_AGG(ARRAY[ranking::text, description] ORDER BY ranking) FROM interest_scale) AS interest,
expertisebot-ben->   (SELECT ARRAY_AGG(ARRAY[ranking::text, description] ORDER BY ranking) FROM experience_scale) AS experience;
                                                         QUERY PLAN
----------------------------------------------------------------------------------------------------------------------------
 Result  (cost=53.98..53.99 rows=1 width=0) (actual time=0.049..0.049 rows=1 loops=1)
   InitPlan 1 (returns $0)
     ->  Aggregate  (cost=26.98..26.99 rows=1 width=36) (actual time=0.031..0.031 rows=1 loops=1)
           ->  Seq Scan on interest_scale  (cost=0.00..19.70 rows=970 width=36) (actual time=0.005..0.008 rows=5 loops=1)
   InitPlan 2 (returns $1)
     ->  Aggregate  (cost=26.98..26.99 rows=1 width=36) (actual time=0.013..0.013 rows=1 loops=1)
           ->  Seq Scan on experience_scale  (cost=0.00..19.70 rows=970 width=36) (actual time=0.001..0.003 rows=5 loops=1)
 Planning time: 0.092 ms
 Execution time: 0.090 ms
(9 rows)
```

```
expertisebot-ben=> EXPLAIN ANALYZE WITH interest_agg AS (
expertisebot-ben(>   SELECT
expertisebot-ben(>     ARRAY_AGG(ARRAY[ranking::text, description] ORDER BY ranking) AS interest
expertisebot-ben(>   FROM interest_scale
expertisebot-ben(> ), experience_agg AS (
expertisebot-ben(>   SELECT
expertisebot-ben(>     ARRAY_AGG(ARRAY[ranking::text, description] ORDER BY ranking) AS experience
expertisebot-ben(>   FROM experience_scale
expertisebot-ben(> )
expertisebot-ben-> SELECT * FROM interest_agg, experience_agg;
                                                         QUERY PLAN
----------------------------------------------------------------------------------------------------------------------------
 Nested Loop  (cost=53.98..54.03 rows=1 width=64) (actual time=0.113..0.115 rows=1 loops=1)
   CTE interest_agg
     ->  Aggregate  (cost=26.98..26.99 rows=1 width=36) (actual time=0.086..0.087 rows=1 loops=1)
           ->  Seq Scan on interest_scale  (cost=0.00..19.70 rows=970 width=36) (actual time=0.005..0.007 rows=5 loops=1)
   CTE experience_agg
     ->  Aggregate  (cost=26.98..26.99 rows=1 width=36) (actual time=0.021..0.021 rows=1 loops=1)
           ->  Seq Scan on experience_scale  (cost=0.00..19.70 rows=970 width=36) (actual time=0.003..0.004 rows=5 loops=1)
   ->  CTE Scan on interest_agg  (cost=0.00..0.02 rows=1 width=32) (actual time=0.088..0.089 rows=1 loops=1)
   ->  CTE Scan on experience_agg  (cost=0.00..0.02 rows=1 width=32) (actual time=0.021..0.022 rows=1 loops=1)
 Planning time: 0.445 ms
 Execution time: 0.215 ms
(11 rows)
```